content/flows/v1/check.ww.en.yaml

```diff
@@ -0,0 +1,32 @@
+type: LinearCarousel
```

Comment:
rename file to check-your-health.ww.en.yaml

Reply:
done